.travis.yml

```diff
@@ -23,6 +23,8 @@ matrix:
       env: TOXENV=py36
     - python: 3.6
       env: TOXENV=docs
+    - python: 3.7-dev
```

Comment:
I think now, as 3.7 is released, it may be better to use Python 3.7 relase (though it may be not straightforward); according to https://github.com/travis-ci/travis-ci/issues/9815 3.7-dev is an old beta.